src/librustdoc/html/render.rs

```diff
@@ -270,6 +270,7 @@ pub struct RenderInfo {
     pub deref_trait_did: Option<DefId>,
     pub deref_mut_trait_did: Option<DefId>,
     pub owned_box_did: Option<DefId>,
+    pub output_format: Option<OutputFormat>,
```

Comment:
I think the `output_format` field would fit better on `DocContext` than `RenderInfo`.

Reply:
It's impacting the rendering so from my point of view, it fits better here. :)